roles/splunk_search_head/tasks/search_head_clustering.yml

```diff
@@ -56,7 +56,7 @@
     splunk_instance_address: "{{ groups['splunk_deployer'][0] }}"
 
 - name: Boostrap SHC captain
-  command: "{{ splunk.exec }} bootstrap shcluster-captain -servers_list '{% for host in groups['splunk_search_head'] %}{{ cert_prefix }}://{{ host }}:{{ splunk.svc_port }}{% if not loop.last %},{% endif %}{% endfor %}' -auth '{{ splunk.admin_user }}:{{ splunk.password }}'"
+  command: "{{ splunk.exec }} bootstrap shcluster-captain -servers_list '{% for host in groups['splunk_search_head'] %}{{ cert_prefix }}://{{ host }}{{ host_fqdn_suffix }}:{{ splunk.svc_port }}{% if not loop.last %},{% endif %}{% endfor %}' -auth '{{ splunk.admin_user }}:{{ splunk.password }}'"
```

Comment:
@mikedickey 
as the host names passed in by orca are in short name pattern, I agree with you to remove "suffix" but before that we should make sure all search head members passed in are with FQDN host name (which could take more effort in orca because we can't guarantee there is no implicit side effect for QA tests).

for init shc task, I suggest use "splunk.host_name" which can be initialized by:
   "**defaultVars["splunk"]["hostname"] = os.environ.get('SPLUNK_HOSTNAME', socket.getfqdn())**"

finally the host names referenced by ansible playbook in search head cluster init and bootstrap tasks are in same FQDN pattern (the /etc/hosts needs update also)